package.json

```diff
@@ -82,5 +82,9 @@
     "versionCompatibility": {
       "ember": ">=3.4"
     }
+  },
+  "volta": {
+    "node": "12.22.10",
+    "yarn": "1.22.17"
```

Comment:
I think Volta makes sense but would move this into a separate PR as it's unrelated to the ESLint fix. We'd also want to have Volta on CI then so it will actually use the right Node/Yarn versions there as well.

Reply:
oh no! sorry about that, intended to stash that change and not push 🙈

Reply:
fixed!